packages/rx_bloc/pubspec.yaml

```diff
@@ -1,23 +1,27 @@
 name: rx_bloc
 description: A Flutter package that helps implementing the BLoC (Business Logic Component) Design Pattern using the power of reactive streams.
-version: 2.0.2
+version: 3.0.0
 repository: https://github.com/Prime-Holding/rx_bloc/tree/master/packages/rx_bloc
 issue_tracker: https://github.com/Prime-Holding/rx_bloc/issues
 homepage: https://www.primeholding.com
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.13.0 <3.0.0"
```

Comment:
@DDavidPrime, do we need to constraint the devs to use 2.13.0, or rx_bloc will be running fine with 2.12 as well?